pogam/scrapers/seloger.py

```diff
@@ -41,7 +45,26 @@
     )""",
     re.UNICODE | re.VERBOSE,
 )
+def map_cp_to_ci (cp: str) -> str:
+    """ 
+        Convert CP to CI as seloger use CI by default
+
+    Args:
+        cp: code postal.
+    Returns:
+        a string of the corresponding code insee recognised by seloger
+    """
+    url = f"https://autocomplete.svc.groupe-seloger.com/api/v2.0/auto/complete/fra/63/10/8/SeLoger?text={cp}"
+    response = requests.get(url)
+    cities = response.json()
+    try:
+        ci=cities[0]['Params']['ci']
+    except Indexerror:
+        msg = f"Unknown post code f{cp}"
+        logger.debug(msg)
```

Comment:
I don't think you need to log this message, because you will see it when your program raises the exception in the next line anyways.